### ISSUE TITLE
Fix compatibility issues with Vuforia: GazeTransform initialization.

### ISSUE THREAD

**Issue by moisoiu** [Microsoft/HoloToolkit-Unity#524]
**What I'm doing**
- I'm using Vuforia framework in combination with Hololens Toolkit and for some reason the DefaultCursor.prefab that I'm using (or any other cursors) the position is set somewhere else.

**How did I found/investigated**
- At first I thought that Cursor.cs would be the main cause of setting the position different but after further investigations I found out that in the GazeManager.cs  the` GazeTransform` receives `Camera.MainTransform` at  `protected override void Awake()` and this sets the position a bit different (moves from center)

**The fix**
- The fix is easy, I've moved `GazeTransform` from `Awake` to `Start` method and everything is working correctly right now.

I would like to know if i can make a pull request regarding to issue.

**Comment by HodgsonSDAS** [comment]
Let me investigate this a bit and get back to you.

**Comment by moisoiu** [comment]
Yes, and everything was working fine.

Regarding to any other scripts, I'm almost sure that Vuforia framework with **ARCamera.prefab** or **VuMark.prefab** might have an impact on changing the position, because If I don't use it (at all, removing the .dll so it doesn't load at startup) the position is staying in the middle (correctly)

I've did the PR [here](https://github.com/Microsoft/HoloToolkit-Unity/pull/528)

**Comment by HodgsonSDAS** [comment]
Seems like a pretty simple fix. I'll review it and get back to you.

**Comment by moisoiu** [comment]
Alright, thanks.

I think the reason why it fixes the problem, it's that when Vuforia Framework -> `ARCamera.prefab`  is setup to use Hololens it requests an Camera object to work. And maybe `ARCamera.prefab` gets another `Camera `and that one sometimes might change the position (or add a different position).

**Comment by HodgsonSDAS** [comment]
Hmm, seems troubling. I hope that's the only collision in the plugins.

**Comment by moisoiu** [comment]
Have fun !

For the moment it's a bit challenging, but in time (as more and more people start to use hololens) it's going to get much more easier. 

P.S 
If you want to play with Vumarks from Vuforia, before anything don't forget to refill your coffee cup :) 

Regards

**Comment by HodgsonSDAS** [comment]
I'm going to keep this open until the PR is merged.

**Comment by NoahMilam** [comment]
Hello,
I was just dealing with the same issue and Im not sure if it is the fix but it seem to work for me. I disable the camera In the ARCamera and it worked. The reason I did that is because in their sample project the have it disabled. Let me know If that is helpful or if I am doing something wrong.

**Comment by moisoiu** [comment]
By disabling the camera in ARCamera you're saying the` ARCamera.prefab` or `Vuforia Behaviour`? 

Because from what I've investigated if you checked **Delayed Initializatior** or even better disable it and instantiate from code the VuMark will not work. 

The ARCamera.prefab is the main thing in Vuforia to make everything regarding to VuMark, Image Target working (of course also license key)

But I have to admit, losing the cursor it happen to me once more, the only thing I did was adding a new script to a different gameobject that wasn't having any relation with ARCamera or HololensCamera and the cursor start to lose the position. **The fix** in my case for that was UNDO the .scene and everything came back to normal.
